### PR TITLE
updated headers and subway page

### PR DESCRIPTION
**Pull request by rrmatthews**
These changes set up the introduction page as well as various headers.

## Screenshot
localhost_3001_education_complaint-tool_form_introduction

### CODE REVIEW

**Review by rianfowler — APPROVED** [review]

**Review by erikphansen — APPROVED** [review]
Just a couple copy nits, otherwise looks good to me. ~~But you'll probably want another +1 from someone else as well.~~

**Review by cehsu — APPROVED** [review]
LGTM, pending @peggygannon's approval.

File: src/applications/edu-benefits/complaint-tool/containers/IntroductionPage.jsx
Comment (rrmatthews) [reply src/applications/edu-benefits/complaint-tool/containers/IntroductionPage.jsx]: This code was just a template where I changed some basic text so I'm not exactly sure why it's there but I thought it would be best to leave it alone.

File: content/pages/education/complaint-tool/form.md
Comment (erikphansen) [inline content/pages/education/complaint-tool/form.md]: I'm always a little wary of using characters outside of the ASCII set directly in HTML. But if the curly single apostrophe works, this ® symbol should work as well. Is this a normal practice.

Otherwise we could use `&#174` instead.

**Review by erikphansen — APPROVED** [review]

**Comment by rrmatthews** [comment]
@lalexanderson-dsva @peggygannon @cehsu This PR is responsible for the content changes and header for the tool. I know there are about to be changes made around incorporating the language of "feedback" instead of complaint throughout, but is it possible to move forward with these changes so we can merge this content PR with [the PR relating to the first page of the tool](https://github.com/department-of-veterans-affairs/vets-website/pull/8013)? Or do i need to hold off until everything is finalized?

**Comment by cehsu** [comment]
Thanks @rrmatthews , it makes sense to merge what is certain and create tickets for outstanding items for now, does that sound good @lalexanderson-dsva @peggygannon ?

File: content/pages/education/complaint-tool/form.md
Comment (cehsu) [inline content/pages/education/complaint-tool/form.md]: Should this be "Feedback" rather than "Complaint" now?

File: src/applications/edu-benefits/complaint-tool/containers/IntroductionPage.jsx
Comment (cehsu) [inline src/applications/edu-benefits/complaint-tool/containers/IntroductionPage.jsx]: typo: "receive"

File: content/pages/education/complaint-tool/form.md
Comment (peggygannon) [reply content/pages/education/complaint-tool/form.md]: @cehsu Yes, the updated title is GI Bill School Feedback Tool.

File: src/applications/edu-benefits/complaint-tool/containers/IntroductionPage.jsx
Comment (peggygannon) [reply src/applications/edu-benefits/complaint-tool/containers/IntroductionPage.jsx]: @cehsu Yes, we are currently changes all instances of "complaint" with "feedback". I'm currently looking at the updated prototype.. Our chances of error are higher when you have to update copy in two places... (prototype and coded files). Can we finalize the prototype and then create the coded files?

**Review by peggygannon — COMMENTED** [review]
@cehsu the title is not correct. I'm currently looking at the updated Figma prototype. Can we wait to create coded files until the prototype is finalized?

**Comment by rrmatthews** [comment]
Closing as we wait for further content changes

**Comment by cehsu** [comment]
@peggygannon indicated [here](https://github.com/department-of-veterans-affairs/vets-website/pull/8064#pullrequestreview-140543321) that we should replace "Complaint" with "Feedback" in the breadcrumb.

**Comment by rrmatthews** [comment]
@peggygannon @cehsu change is made, just forgot to update the screenshot but I'll do that right now

**Comment by cehsu** [comment]
Thanks @peggygannon and @rrmatthews, @rrmatthews could you please provide an updated screenshot if you have not already done so for @peggygannon to review, and ping her when it is ready for her to take a look?

**Comment by rrmatthews** [comment]
@peggygannon  @cehsu  Screenshot is updated

**Review by erikphansen — APPROVED** [review]

**Comment by peggygannon** [comment]
@cehsu @rrmatthews Please see below for additional edits to this page:

--Do we need to have the registered symbol on GI Bill in the breadcrumb? 

Please drop in this new UPDATED INTRO:
You can submit feedback to us if your GI Bill school isn’t following the Principles of Excellence guidelines or if you have any other concerns or issues you’d like to raise with us. The Principles of Excellence program requires schools to follow certain guidelines in order to get federal funding through the GI Bill. 
You can choose to submit feedback anonymously or on behalf of someone else. Any feedback sent in anonymously isn't shared with schools or employers. 
Get started right now by filling out a short form. Follow the steps below.

Please delete: **Follow the steps below to submit your feedback**


Under **VA Review**
Please update the email address to: process.vbavaco@va.gov 

Delete this sentence: "Please add this to your email contact list so you can respond to messages that may need a quick response from you."

Please run in this sentence to the one before it: "If we need to get in touch with you, we'll contact you form an email that will look like this: process.vbavaco@va.gov."

Under *School Follow Up**
please add this sentence "In some cases, the school may get in touch with you directly to discuss your feedback."   AFTER ..."if you think their response resolves your issue."

**Comment by cehsu** [comment]
Thanks @peggygannon, updates made, kindly refer to the new screenshot below. Also in regards to your question: 

> --Do we need to have the registered symbol on GI Bill in the breadcrumb?

This may be a question for @miamay345 @emilywaggoner or @lalexanderson-dsva ?

localhost_3001_education_complaint-tool_form_introduction 1

**Review by peggygannon — APPROVED** [review]
@cehsu OK, looks good. I just went in and made one more edit from the stakeholders (changing "application" to "form")... some of them are still looking at it.

**Comment by cehsu** [comment]
Thanks @peggygannon, I see the breadcrumb includes the registered symbol in the design and have included it and updated the screenshot above.

**Comment by peggygannon** [comment]
Thanks, @cehsu Screenshot looks good.

**Comment by cehsu** [comment]
@peggygannon, the following sentence (which appears on the confirmation page) looks a little unclear, is this the final content?

> Feedback that isn't related to VA education benefits may be sent to another agency for review.

**Pull request merged**
